spec/rspec/core/bisect/runner_spec.rb

```diff
@@ -224,7 +225,7 @@ def repro_command_from(ids)
 
       it "runs the suite with the locations from the original CLI args" do
```

Comment:
This doc string is a bit confusing since the expectation isn't about the locations anymore.  Can you change it to something like `it "runs the suite with the original CLI args" do`?